pages/docs/reference/js-to-kotlin-interop.md

```diff
@@ -30,21 +26,20 @@ alert(myModule.foo());
 ```
 </div>
 
-This is not applicable when you compile your Kotlin module to JavaScript module (see [JavaScript Modules](js-modules.html) for more information on this).
-In this case there won't be a wrapper object, instead, declarations will be exposed as a JavaScript module of a corresponding kind. For example,
-in case of CommonJS you should write:
-
+This is not applicable when you compile your Kotlin module to JavaScript modules like UMD (which is the default setting for both `browser` and `nodejs` targets), CommonJS or AMD. In this case, your declarations will be exposed in the format specified by your chosen JavaScript module system. When using UMD or CommonJS, for example, your call site could look like this:
+<!-- TODO: IS THIS STILL CORRECT? -->
```

Comment:
Why do you doubt it? It looks correct to me modulo that foo should be explicitly exported in IR BE.

Reply:
AFAIK This was a remnant of the discussion about `commonjs` vs `commonjs2`, which changed the export behavior slightly. I've removed the TODO.